src/actions/vstudio/vs2010_vcxproj.lua

```diff
@@ -1104,6 +1105,13 @@
 	end
 
 
+	function m.linkObjects(fcfg, condition)
+		if fcfg.linkbuildoutputs == false then
+			m.element("LinkObjects", condition, "false")
```

Comment:
Should this be:
```lua
if fcfg.linkbuildoutputs then
  m.element("LinkObjects", condition, fcfg.linkbuildoutputs)
```

Generally, an element is emit when the user specifies a value for it, regardless of whether it's the default or not.